### ISSUE TITLE
ERROR Unexpected exception: Command 'fairseq-train /content/mmt/runtime/DEBUG/tmp/training/data_generated

### ISSUE THREAD

**Issue by aaraki** [modernmt/modernmt#610]
Hello,I'm using colab.I'm trying to execute
"!mmt/mmt create ja en -e mmt_ja-en mmt/examples/kftt/train --train-steps 10 --engine DEBUG --debug",
then I get the following error.Could you teach me the solution?
when I set the "--no-epoch-checkpoints" parameter,I get the same error.

(4/4) Training neural model
(1/2) Train neural network...
ERROR Unexpected exception: Command 'fairseq-train /content/mmt/runtime/DEBUG/tmp/training/data_generated --save-dir /content/mmt/runtime/DEBUG/tmp/training/_temp_train/nn_model --task mmt_translation --user-dir /content/mmt/build/mmt-4.5.4.jar/mmt --share-all-embeddings --no-progress-bar --tensorboard-logdir /content/mmt/runtime/DEBUG/tmp/training/_temp_train/tensorboard_logdir --dataset-impl mmap --max-update 10 --debug --arch transformer_mmt_base --clip-norm 0.0 --label-smoothing 0.1 --attention-dropout 0.1 --dropout 0.3 --weight-decay 0.0 --criterion label_smoothed_cross_entropy --optimizer adam --adam-betas (0.9, 0.98) --log-interval 100 --lr 0.0005 --lr-scheduler inverse_sqrt --min-lr 1e-09 --warmup-init-lr 1e-07 --warmup-updates 4000 --max-tokens 3072 --update-freq 4 --save-interval-updates 1000 --keep-interval-updates 10 --keep-last-epochs 10' failed with exit code 2
